yt/visualization/volume_rendering/lens.py

```diff
@@ -180,8 +180,8 @@ def _get_sampler_params(self, camera, render_source):
         north_vec = camera.unit_vectors[1]
         normal_vec = camera.unit_vectors[2]
 
-        px = np.mat(np.linspace(-0.5, 0.5, camera.resolution[0]))
-        py = np.mat(np.linspace(-0.5, 0.5, camera.resolution[1]))
+        px = np.linspace(-0.5, 0.5, camera.resolution[0])
```

Comment:
I am not terribly comfortable with this, but my limited testing has suggested it should produce identical results.

Reply:
Same here. np.mat is deprecated and I wasn't able to find a reason for using it here...

Reply:
I _think_ this is equivalent, but if we want to be 100% sure, we can do:
```suggestion
        px = np.linspace(-0.5, 0.5, camera.resolution[0])[np.newaxis, :]
```
here and for `py` below.
which will create an array with shape `(1, camera.resolution[0])`.